filesystem/config.pan.new

```diff
@@ -0,0 +1,804 @@
+# Template allow to configure partitionning with quite a lot of flexibility,
```

Comment:
partition~~n~~ing

So which template do you recommend using? Why not just replace `filesystem/config.pan`?

Whole template could do with `nlist` → `dict` and removal of `include { }`.